apps/remote_control/lib/lexical/remote_control/code_intelligence/ast.ex

```diff
@@ -52,13 +52,26 @@ defmodule Lexical.RemoteControl.CodeIntelligence.Ast do
   @spec expand_aliases(
           document :: Document.t(),
           position :: Position.t(),
-          module_aliases :: list()
+          module_aliases :: list(),
+          already_in_container? :: boolean()
```

Comment:
I think this function is not just used by the struct fields, so I also put the `already_in_container` arguments here too.